week-3/Homework/mandatory/6-lane-names.js

```diff
@@ -2,11 +2,19 @@
   You are given a list of some London street names.
   
   Write a function that will return all street names which contain 'Lane' in their name.
+  
 */
-
-function getLanes() {
+//hint: string and array methods that could be helpful (indexOf, filter)
+function getLanes(arr) {
+    var newArr=arr.filter(key=>key.includes('Lane'));
```

Comment:
```suggestion
    var newArr=arr.filter(key => key.includes('Lane'));
```
Good use of `includes()` - think about a different parameter name other than `key`